src/VectorBasemapLayer.js

```diff
@@ -65,6 +65,19 @@ export var VectorBasemapLayer = VectorTileLayer.extend({
     } else {
       styleUrl = getBasemapStyleUrl(this.options.key, this.options.apikey);
     }
+    // show error warning on successful response
+    fetch(styleUrl)
```

Comment:
Can we avoid sending an additional fetch request??

I have been checking the previous error message:
<img width="919" alt="Screenshot 2024-10-30 at 08 35 38" src="https://github.com/user-attachments/assets/3f6ee055-c83d-4025-b643-12ec756d14c9">

But it doesn't give us the information we need from the REST API, right? I have been checking [other events](https://maplibre.org/maplibre-gl-js/docs/API/type-aliases/MapEventType/#load) unsuccessfully.

The [style.ts](https://github.com/Esri/esri-leaflet-vector/pull/227#issuecomment-2443938163) is part of MapLibre library so we can not access to any.

// cc: @gavinr-maps @gowin20 any clue on how we can get that REST response?

Reply:
@Kizito007 I discussed this today with @gavinr-maps and @hhkaos. We are ok with the additional `fetch` request but only if the `options.version` is `1` since auth errors with the version 2 service will get covered by the `error` event above.

Reply:
Just to confirm @patrickarlt are there any variations from version `1` such as `1.1.0 or 1.0.1` so I can safely check if `options.version === 1`

Reply:
Yes, version is only `1` or `2` - see here: https://github.com/Esri/esri-leaflet-vector/blob/c751bafe722c14b516000d653f6687b62ce4de3a/src/VectorBasemapLayer.js#L17-L18

Reply:
Thanks for the feedback bosses @gavinr-maps @hhkaos @patrickarlt 

I just pushed the update.